### PR TITLE
Maintenance B

### PR DESCRIPTION
**Pull request by marchcat**
as of = 2024-06-21 for RC deploy
build = https://github.com/secondlife/viewer/releases/tag/Second_Life_Release%23a65bc46-b
cohort = Maintenance-B
deployed = https://github.com/secondlife/viewer/releases/tag/Second_Life_Release%23a65bc46-b
desired = 2000

relnotes:
### Maintenance B: Bellusaurus

The name Bellusaurus means "beautiful lizard" which is an acclamation to this herbivores graceful form[.](https://wiki.secondlife.com/wiki/File:Bellusaurus.webp) It is believed that this small dinosaur was quite social and lived in herds during the Middle Jurassic period.

This beautifully named release brings many small beautiful updates including the renaming of a certain imposter setting in Advanced Graphics Preferences. The setting "Max # of non-imposters" will henceforth be known as "Maximum number of animated avatars" and will also be located on the basic Graphics Preferences tab in Preferences.

Additionally, some other small fixes have been added to this release which are listed below.

### CODE REVIEW

**Comment by marchcat** [comment]
build = https://github.com/secondlife/viewer/releases/tag/Second_Life_Release%23bcb5210-b

**Comment by marchcat** [comment]
build = https://github.com/secondlife/viewer/releases/tag/Second_Life_Release%238701e09-b

File: indra/newview/app_settings/shaders/class3/deferred/materialF.glsl [hunk 1]
```diff
@@ -53,7 +53,7 @@ out vec4 frag_color;
 float sampleDirectionalShadow(vec3 pos, vec3 norm, vec2 pos_screen);
 #endif
 
-void sampleReflectionProbesLegacy(inout vec3 ambenv, inout vec3 glossenv, inout vec3 legacyenv,
+void sampleReflectionProbesLegacy(out vec3 ambenv, out vec3 glossenv, out vec3 legacyenv,
```
[the remark this answers is not in the document]
Comment (yaynstuff) [reply indra/newview/app_settings/shaders/class3/deferred/materialF.glsl:56]: Giving pink avatars for me on linux.... Reverting it fixes the issue

[the remark this answers is not in the document]
Comment (yaynstuff) [reply indra/newview/app_settings/shaders/class3/deferred/materialF.glsl:56]: Ahh I just was sent this: https://github.com/secondlife/viewer/commit/72730609f371ae0417a2d117fdc9d805ce8858fb -- so we should be good

**Comment by marchcat** [comment]
build = https://github.com/secondlife/viewer/releases/tag/Second_Life_Release%233c833ce-b

**Comment by AtlasLinden** [comment]
| Maint B RC 7.1.7.8820696922 | Result | Comment | 
|---|---|---|
| secondlife/iqa#209 Acceptance, Integration & Additional tests | Passed | Passed |
| **Overall Result** | **Passed** | **OK to ship** | 

**NOTES**
No blockers

Anti-virus false positive report:
| File | Security vendor analysis |
|---|---|
| slplugin.exe | Bkav Pro - W64.AIDetectMalware <br> MaxSecure - Trojan.Malware.300983.susgen <br> SecureAge - Malicious |
| SLVersionChecker.exe | Antiy-AVL - Trojan/Win32.Wacatac <br> SecureAge - Malicious |
| slvoice.exe | Bkav Pro - W64.AIDetectMalware |
| uninst.exe | Bkav Pro - W32.AIDetectMalware |

**Review by kylelinden — APPROVED** [review]
Maint B RC approved
viewer = 7.1.7.8820696922
build = https://github.com/secondlife/viewer/releases/tag/Second_Life_Release%233c833ce-b
cohort = Maintenance-B
desired = 3000

**Comment by marchcat** [comment]
игшдв = https://github.com/secondlife/viewer/releases/tag/Second_Life_Release%234fbb190-b

**Comment by AtlasLinden** [comment]
| Maint B RC 7.1.8.9130881608 | Result | Comment | 
|---|---|---|
| secondlife/iqa#237 Acceptance, Smoke & Additional tests | Passed | Passed |
| **Overall Result** | **Passed** | **OK to ship** | 

**NOTES**
No blockers

Anti-virus false positive report:
| File | Security vendor analysis |
|---|---|
| slplugin.exe | MaxSecure - Trojan.Malware.300983.susgen <br> SecureAge - Malicious |
| SLVersionChecker.exe | SecureAge - Malicious |

**Review by kylelinden — APPROVED** [review]
Maint B RC approved.
viewer =  7.1.8.9130881608
build = https://github.com/secondlife/viewer/releases/tag/Second_Life_Release%234fbb190-b
cohort = Maintenance-B
desired = 3000

**Comment by vir-linden** [comment]
deployed = https://github.com/secondlife/viewer/releases/tag/Second_Life_Release%234fbb190-b

**Comment by marchcat** [comment]
build = https://github.com/secondlife/viewer/releases/tag/Second_Life_Release%23847a845-b

**Comment by marchcat** [comment]
build = https://github.com/secondlife/viewer/releases/tag/Second_Life_Release%23a65bc46-b

**Comment by AtlasLinden** [comment]
| Maint B RC 7.1.9.9555137545 | Result | Comment | 
|---|---|---|
| secondlife/iqa#263 Acceptance, Integration & Additional tests | Passed | Passed |
| **Overall Result** | **Passed** | **OK to ship** | 

**NOTES**
No blockers but https://github.com/secondlife/viewer/issues/1805 was filed

Anti-virus false positive report:
Files rescanned from IQA
| File | Security vendor analysis |
|---|---|
| SecondLife.exe | MaxSecure - Win.MxResIcn.Heur.Gen |
| slplugin.exe | MaxSecure - Trojan.Malware.300983.susgen <br> SecureAge - Malicious <br> Skyhigh (SWG) -  BehavesLike.Win64.CoinMiner.hh |
| SLVersionChecker.exe | MaxSecure - Win.MxResIcn.Heur.Gen <br> SecureAge - Malicious |
| SLVoice.exe | MaxSecure - Win.MxResIcn.Heur.Gen |
| dullahan_host.exe | MaxSecure - Win.MxResIcn.Heur.Gen |

**Review by kylelinden — APPROVED** [review]
Maint B RC approved
viewer = 7.1.9.9555137545
build = https://github.com/secondlife/viewer/releases/tag/Second_Life_Release%23a65bc46-b
cohort = Maintenance-B
desired = 3000

**Comment by nat-goodspeed** [comment]
deployed = https://github.com/secondlife/viewer/releases/tag/Second_Life_Release%23a65bc46-b

**Comment by Dan-Linden** [comment]
desired = 2000
(because we need WebRTC cohort to fill faster)

**Comment by marchcat** [comment]
build = https://github.com/secondlife/viewer/releases/tag/Second_Life_Release%23206bdc9-b

**Comment by marchcat** [comment]
build = https://github.com/secondlife/viewer/releases/tag/Second_Life_Release%23ea8d7cb-b

**Comment by marchcat** [comment]
build = https://github.com/secondlife/viewer/releases/tag/Second_Life_Release%23b111dce-maint-b-2024-09-19

**Review by brad-linden — APPROVED** [review]

**Comment by AtlasLinden** [comment]
| Maint B RC 7.1.11.10945332304 | Result | Comment | 
|---|---|---|
| secondlife/iqa#347 Acceptance, Integration & Additional tests | Passed | Passed |
| **Overall Result** | **Passed** | **OK to ship** | 

**NOTES**
No blockers.
Odd grid selector bugs: 
- https://github.com/secondlife/viewer/issues/2697
- https://github.com/secondlife/viewer/issues/2698

Anti-virus false positive report:
| File | Security vendor analysis |
|---|---|
| slplugin.exe | Bkav Pro - W64.AIDetectMalware <br> MaxSecure - Trojan.Malware.300983.susgen <br> SecureAge - Malicious |
| SLVersionChecker.exe | SecureAge - Malicious |

**Pull request merged**